### PR TITLE
Update add-accounts.json

### PR DESCRIPTION
**Pull request by jimmy-tudeski**
Polish translation for polkadot.js.apps

### CODE REVIEW

**Review by jacogr — APPROVED** [review]
Thank you :)

**Comment by jimmy-tudeski** [comment]
Hey, please check the full translation available :)

File: packages/apps/public/locales/pl/add-accounts.json [hunk 1]
```diff
@@ -1,340 +1,340 @@
...
-  "Pending call hashes": "Pending call hashes",
-  "Please make sure to save this file in a secure location as it is required, together with your password, to restore your account.": "Please make sure to save this file in a secure location as it is required, together with your password, to restore your account.",
-  "Prev": "Prev",
-  "Private Key": "Private Key",
-  "Provide judgement": "Provide judgement",
-  "Provide the account QR from the module/external application for scanning. Once detected as valid, you will be taken to the next step to add the account to your list.": "Provide the account QR from the module/external application for scanning. Once detected as valid, you will be taken to the next step to add the account to your list.",
-  "Proxied": "Proxied",
-  "Proxied account has no owned proxies": "Proxied account has no owned proxies",
-  "Proxy overview": "Proxy overview",
+  "Normal transfer without keep-alive checks": "Zwykły transfer bez funkcji \"keep-alive"\.",
```
Comment (jacogr) [inline packages/apps/public/locales/pl/add-accounts.json:102]:
```suggestion
  "Normal transfer without keep-alive checks": "Zwykły transfer bez funkcji \"keep-alive\".",
```

File: packages/apps/public/locales/pl/add-accounts.json [hunk 2]
```diff
@@ -1,340 +1,340 @@
...
+  "We will provide you with a generated backup file after your account is created. As long as you have access to your account you can always download this file later by clicking on \"Backup\" button from the Accounts section.": "Po utworzeniu konta dostarczymy Ci wygenerowany plik kopii zapasowej. Jeśli masz dostęp do swojego konta, zawsze możesz pobrać ten plik później, klikając przycisk \"Kopia zapasowa\" w sekcji Konta.",
+  "With the keep-alive option set, the account is protected against removal due to low balances.": "Dzięki ustawionej opcji utrzymywania przy życiu konto jest chronione przed usunięciem z powodu niskiego salda.",
+  "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. An optional \"///<password>\" can be used with a mnemonic seed, and may only be specified once.": "Możesz ustawić niestandardową ścieżkę wyprowadzania dla tego konta, używając następującej składni \"/<soft-key>//<hard-key>\".\"/<soft-key>\" i \"//<hard-key>\" mogą być powtarzane i mieszane.", \"/// <hasło>\" można użyć z mnemonic seed i można je podać tylko raz.",
```
Comment (jacogr) [inline packages/apps/public/locales/pl/add-accounts.json:233]:
```suggestion
  "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. An optional \"///<password>\" can be used with a mnemonic seed, and may only be specified once.": "Możesz ustawić niestandardową ścieżkę wyprowadzania dla tego konta, używając następującej składni \"/<soft-key>//<hard-key>\".\"/<soft-key>\" i \"//<hard-key>\" mogą być powtarzane i mieszane. \"/// <hasło>\" można użyć z mnemonic seed i można je podać tylko raz.",
```

File: packages/apps/public/locales/pl/add-accounts.json [hunk 3]
```diff
@@ -1,340 +1,340 @@
...
+  "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>///<password>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`.": "Możesz ustawić niestandardową ścieżkę wyprowadzania dla tego konta, używając następującej składni \"/<soft-key>//<hard-key>///<password>\".\"/<soft-key>\" i \"// <hard-key>\" mogą być powtarzane i mieszane..",
+  "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>///<password>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. The \"///password\" is optional and should only occur once.": "Możesz ustawić niestandardową ścieżkę wyprowadzania dla tego konta, używając następującej składni \"/<soft-key>//<hard-key>///<password>\".\"/<soft-key>\" i \"//<hard-key>\" mogą być powtarzane i mieszane ". \"///hasło\" jest opcjonalne i powinno wystąpić tylko raz.",
```
Comment (jacogr) [inline packages/apps/public/locales/pl/add-accounts.json:235]:
```suggestion
  "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>///<password>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. The \"///password\" is optional and should only occur once.": "Możesz ustawić niestandardową ścieżkę wyprowadzania dla tego konta, używając następującej składni \"/<soft-key>//<hard-key>///<password>\".\"/<soft-key>\" i \"//<hard-key>\" mogą być powtarzane i mieszane. \"///hasło\" jest opcjonalne i powinno wystąpić tylko raz.",
```

File: packages/apps/public/locales/pl/add-accounts.json [hunk 4]
```diff
@@ -1,340 +1,340 @@
...
+  "Total amount of fund that will be reserved. These funds are returned when the identity is cleared": "Całkowita kwota funduszu, który zostanie zarezerwowany. Środki te są zwracane po skasowaniu tożsamości",
+  "Transfer the full account balance, reap the sender": "Przenieś pełne saldo konta, zamykając je",
+  "Transfer with account keep-alive checks": "Przeew z bieżacym sprawdzeniem stanu transackji i poprawności stanu po dokonaniu stransakcji",
+  "Type here what you would like your address to contain. This tool will generate the keys and show the associated addresses that best match your search. You can use \"?\" as a wildcard for a character.": "Wpisz tutaj, co chcesz, aby zawierał Twój adres. To narzędzie wygeneruje klucze i pokaże powiązane adresy, które najlepiej pasują do Twojego wyszukiwania. Możesz użyć \ "? \" Jako symbolu wieloznacznego dla znaku.",
```
Comment (jacogr) [inline packages/apps/public/locales/pl/add-accounts.json:222]:
```suggestion
  "Type here what you would like your address to contain. This tool will generate the keys and show the associated addresses that best match your search. You can use \"?\" as a wildcard for a character.": "Wpisz tutaj, co chcesz, aby zawierał Twój adres. To narzędzie wygeneruje klucze i pokaże powiązane adresy, które najlepiej pasują do Twojego wyszukiwania. Możesz użyć \"?\" Jako symbolu wieloznacznego dla znaku.",
```

File: packages/apps/public/locales/pl/add-accounts.json [hunk 5]
```diff
@@ -1,340 +1,340 @@
...
+  "There is an existing reference count on the sender account. As such the account cannot be reaped from the state.": "Na koncie nadawcy istnieją referencje. W związku z tym konto nie może być wyczyszczone do zera.",
+  "These are trusted individuals that can verify and approve any recovery actions. With recovery, once the threshold is reached, the funds associated with the account can be moved to a new destination.": "Są to zaufane osoby, które mogą weryfikować i zatwierdzać wszelkie działania naprawcze. Po osiągnieciu progu decyzyjności środki powiązane z kontem mogą zostać przeniesione w nowe miejsce.",
+  "This account has a governance delegation": "To konto ma delegację głosów governance",
+  "This account has {{proxyNumber}} proxy set.": "To konto ma następującą liczbę {{proxyNumber}} ustawionych proxy.",
+  "This account is available on all networks. Ensure you tie it to a specific network via the account options (\"only this network\" option) to limit availability on a single network.": "To konto jest dostępne we wszystkich sieciach. Upewnij się, że łączysz je z określoną siecią za pomocą opcji konta (opcja \„tylko ta sieć\”), aby ograniczyć dostępność w jednej sieci.",
```
Comment (jacogr) [inline packages/apps/public/locales/pl/add-accounts.json:209]:
```suggestion
  "This account is available on all networks. Ensure you tie it to a specific network via the account options (\"only this network\" option) to limit availability on a single network.": "To konto jest dostępne we wszystkich sieciach. Upewnij się, że łączysz je z określoną siecią za pomocą opcji konta (opcja \"tylko ta sieć\”), aby ograniczyć dostępność w jednej sieci.",
```

**Review by jacogr — APPROVED** [review]

**Pull request merged**